include/vrv/horizontalaligner.h

```diff
@@ -75,7 +124,7 @@ class Alignment : public Object {
      */
     ///@{
     Alignment();
-    Alignment(double time, AlignmentType type = ALIGNMENT_DEFAULT);
+    Alignment(const Fraction &time, AlignmentType type = ALIGNMENT_DEFAULT);
```

Comment:
No need to change, but note that `const Fraction &time` is good, but `Fraction time` would work just as well (class contains only two integers) so placing a copy on the stack should not be a problem (and in theory it could be faster since it would not involve setting up pointers behind the scenes).